poa/candidates.go

```diff
@@ -44,8 +44,8 @@ func NewCandidates(list []*authority.Candidate) *Candidates {
 // Copy make a copy.
 func (c *Candidates) Copy() *Candidates {
 	c.referenced = true
-	copy := *c
-	return &copy
+	cpy := *c
```

Comment:
What's the rule behind this?

Reply:
No rule per say, but copy matches the `inbuilt.copy` function.

Reply:
Isn't that strange that it prefers `cpy` over `copy`?

Reply:
Hmm, `copy` is a reserved keyword :-)